spec/ddtrace/workers/runtime_metrics_spec.rb

```diff
@@ -7,12 +7,17 @@
 RSpec.describe Datadog::Workers::RuntimeMetrics do
   subject(:worker) { described_class.new(options) }
 
-  let(:metrics) { instance_double(Datadog::Runtime::Metrics, close: nil) }
+  let(:metrics) { instance_double(Datadog::Core::Runtime::Metrics, close: nil) }
   let(:options) { { metrics: metrics, enabled: true } }
 
   before { allow(metrics).to receive(:flush) }
 
-  after { worker.stop(true, 5) }
+  after { worker.stop(true, 1) }
+
+  shared_context 'short default flush interval' do
+    let(:default_flush_interval) { 0.1 }
+    before { stub_const('Datadog::Workers::RuntimeMetrics::DEFAULT_FLUSH_INTERVAL', default_flush_interval) }
+  end
```

Comment:
Since this is only used once, I suggest inlining it on the test that uses it.

Reply:
Probably will; I added it because I thought multiple tests needed it. Turns out only one did.

Reply:
Fixed in the next PR.